test/qa/test_meshnet.py

```diff
@@ -317,4 +319,167 @@ def test_account_switch():
     # Recover starting state (this is the simplest way)
     teardown_module(None)
     setup_module(None)
-    
\ No newline at end of file
+
+
+def test_invite_send_repeated():
+
+    with lib.Defer(lambda: sh.nordvpn.meshnet.invite.revoke("test@test.com")):
+        process = subprocess.Popen(sh.nordvpn.meshnet.invite.send("test@test.com"), shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
```

Comment:
it might be good to move the entire `send` flow into a helper function.

Reply:
Good idea, made changes accordingly, thanks!